index.html

```diff
@@ -0,0 +1,222 @@
+<!dOCTYPE html>
+<html lang="zxx">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Smashing Magazine design teardown</title>
+    <link href="https://fonts.googleapis.com/css?family=Acme&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Patua+One&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/stylesheets/reset.css">
+    <link rel="stylesheet" href="assets/stylesheets/main.css">
+</head>
+
+<body>
+    <header class="w6">
+        <div>
+            <h1>Smashing <br>
+                Magazine</h1>
+        </div>
+        <nav>
+            <ul>
+                <li>
+                    <h2> Articles</h2>
+                </li>
+                <li>
+                    <h2>BOOKS</h2>
+                </li>
+                <li>
+                    <h2>EVENTS</h2>
+                </li>
+                <li>
+                    <h2>JOBS</h2>
+                </li>
+                <li>
+                    <h2>ME</h2>
+                </li>
+            </ul>
+        </nav>
+        <div class="topics">
+            <h3>TOPICS</h3>
+        </div>
+
+
+    </header>
+
+    <main>
+        <h1 class="miss">
+            <span class=" w7">Don’t Miss These
+                Latest</span><br>
+            <span class=" w7">Article</span>
+        </h1>
+
+        <section class="articles">
+            <h2 class="miss">
+                <span class=" w7">Don’t Miss These
+                    Latest</span><br>
+                <span class=" w7">Article</span>
+            </h2>
+            <section class=" w3">
```

Comment:
Each one of the four main article must have an image in the top left corner, just like the original page. You don't need to use an actual image with `img` tag here. Create a container (`div`) that represents the image and use a different shade of gray in it.